From: Jens Axboe <axboe@suse.de>
Subject: quiet down ide-cd capacity read
Patch-mainline: 2.6.17-rc1
References: 159002

Some drives like to tell us that the installed medium doesn't support
this command, so quiet it down.

Acked-by: 
Signed-off-by: 

--- linux-2.6.16/drivers/ide/ide-cd.c~	2006-03-21 15:20:34.000000000 +0100
+++ linux-2.6.16/drivers/ide/ide-cd.c	2006-03-21 15:21:00.000000000 +0100
@@ -2142,6 +2142,7 @@
 	req.cmd[0] = GPCMD_READ_CDVD_CAPACITY;
 	req.data = (char *)&capbuf;
 	req.data_len = sizeof(capbuf);
+	req.flags |= REQ_QUIET;
 
 	stat = cdrom_queue_packet_command(drive, &req);
 	if (stat == 0) {
